services/src/main/java/org/apache/druid/cli/CliOverlord.java

```diff
@@ -300,7 +300,7 @@ private void configureRunners(Binder binder)
                 binder,
                 "druid.indexer.runner.type",
                 Key.get(TaskRunnerFactory.class),
-                Key.get(ForkingTaskRunnerFactory.class)
+                Key.get(HttpRemoteTaskRunnerFactory.class)
```

Comment:
With this change, I think there's no need for `druid.indexer.runner.type=httpRemote` in all the bundled configs anymore. Could you try removing that line and see if everything still works OK?

Reply:
Done, thanks!